pcdsdevices/radiation.py

```diff
@@ -0,0 +1,19 @@
+"""
+Radiation Sesnsor Classes.
+
+This Module contains all the classes relating to Radiation instrumentation
+"""
+from ophyd import Component as Cpt
+from ophyd import EpicsSignalRO
+from ophyd.device import Device
+
+
+class HPI6030(Device):
+    """class for reading out 6030 sensor data"""
```

Comment:
Does HPI stand for anything?  It might be nice to have that in the docstring rather than "class".  I end up in pcdsdevices to learn what devices do anyway, so maybe I'm just propagating that 😄

Reply:
yea probably anything is better than class, ill update this.